src/postgres/src/interfaces/libpq/fe-connect.c

```diff
@@ -1132,7 +1132,7 @@ connectOptions2(PGconn *conn)
 	{
 		if (conn->dbName)
 			free(conn->dbName);
-		conn->dbName = strdup(conn->pguser);
+		conn->dbName = strdup("yugabyte");
```

Comment:
Can you also add this to `conninfo_add_defaults()` for consistency? - we are setting other default values like username in that method.

Reply:
@ndeodhar I have added default `dbname` to `conninfo_add_defaults()` as suggested. Thanks!